tests/run_tests.py

```diff
@@ -50,6 +52,7 @@ def plural(num):
 
 def get_validity(sexpected, sresult, soutfile, force_id_mask=False):
     """Decipher whether the output passes, fails, or initializes"""
+    sresult = sresult
```

Comment:
Nope.  That is a left-over from my `decode`/`encode` whack-a-mole.  I need to study the code base a little bit more to efficiently encode things at the beginning and decode at the end to avoid messing up things.